data/validations/itemValidations.js

```diff
@@ -209,6 +209,18 @@ function isValidBid(itemId, bid, userId) {
   }
 }
 
+function isValidPrice(price){
```

Comment:
does this need to be exported or only internal to the file? Sorry, I definitely deleted this on accident

Reply:
no worries, its just called locally within the itemValidations file